content/sensu-go/5.0/api/checks.md

```diff
@@ -94,28 +100,74 @@ output         | {{< highlight shell >}}
 
 ### `/checks` (POST)
 
+#### EXAMPLE {#checks-post-example}
+
+In the following example, an HTTP POST request is submitted to the `/checks` API to create a `check-cpu` check.
+The request includes the check definition in the request body and returns a successful HTTP 200 OK response and the created check definition.
+
+{{< highlight shell >}}
+curl -X POST \
+-H "Authorization: Bearer TOKEN" \
+-H 'Content-Type: application/json' \
+-d '{
+  "command": "check-cpu.sh -w 75 -c 90",
+  "subscriptions": [
+    "linux"
+  ],
+  "interval": 60,
+  "publish": true,
+  "handlers": [
+    "slack"
+  ],
+  "metadata": {
+    "name": "check-cpu",
+    "namespace": "default"
+  }
+}' \
+http://127.0.0.1:8080/api/core/v2/namespaces/default/checks
+
+HTTP/1.1 200 OK
+{
+  "command": "check-cpu.sh -w 75 -c 90",
+  "subscriptions": [
+    "linux"
+  ],
+  "interval": 60,
+  "publish": true,
+  "handlers": [
+    "slack"
+  ],
+  "metadata": {
+    "name": "check-cpu",
+    "namespace": "default"
+  }
+}
+{{< /highlight >}}
+
+#### API Specification {#checks-post-specification}
+
 /checks (POST) | 
 ----------------|------
 description     | Create a Sensu check.
 example URL     | http://hostname:8080/api/core/v2/namespaces/default/checks
-payload         | {{< highlight shell >}}
+example payload | {{< highlight shell >}}
 {
-  "command": "http_check.sh https://sensu.io",
-  "handlers": [
-    "slack"
+  "command": "check-cpu.sh -w 75 -c 90",
+  "subscriptions": [
+    "linux"
   ],
-  "interval": 15,
-  "proxy_entity_name": "sensu.io",
+  "interval": 60,
   "publish": true,
-  "subscriptions": [
-    "site"
+  "handlers": [
+    "slack"
   ],
   "metadata": {
-    "name": "check-sensu-site",
+    "name": "check-cpu",
     "namespace": "default"
   }
 }
 {{< /highlight >}}
+payload parameters | Required check attributes: `command` (string), `subscriptions` (array of strings), `interval` (integer), `publish` (set to `true`), and a `metadata` scope containing `name` (string) and `namespace` (string). For more information about creating checks, see the [check reference][1]. 
```

Comment:
Check interval is a funny requirement. A check essentially requires a schedule, which could be `interval` or `cron`.
```suggestion
payload parameters | Required check attributes: `command` (string), `subscriptions` (array of strings), `interval` (integer) OR `cron` (string), `publish` (set to `true`), and a `metadata` scope containing `name` (string) and `namespace` (string). For more information about creating checks, see the [check reference][1]. 
```